src/com/layout/col/col.js

```diff
@@ -0,0 +1,30 @@
+import {h, Component} from 'preact/preact';
+
+export default class Col extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render({children, flex = 1, flexGrow, flexShrink, flexBasis = 100}) {
+    // let props = Object.assign({}, this.props, {
+    //   flex: undefined,
+    //   flexBasis: undefined,
+    //   flexGrow: undefined,
+    //   flexShrink: undefined
+    // });
+
+    flexGrow = flexGrow != undefined
+      ? flexGrow
+      : flex;
+
+    flexShrink = flexShrink != undefined
+      ? flexShrink
+      : flex;
+
+    return (
+      <div class={cN([this.props.class, "-col"])} style={`flex: ${flexGrow} ${flexShrink} ${flexBasis}%; max-width: ${flexBasis}%;`} {...this.props}>
```

Comment:
Array brackets should be removed from `cN([this.props.class, "-col"])`. 

`this.props` should not be used in `render` (`props` and `state` are always available as arguments).

Reply:
Don't use backtick strings and `${}` syntax (not supported by IE).

Reply:
I was pretty sure buble handled the conversion of template strings to regular strings. I'll look into it and change it if that wasn't the case.

Admittedly I had issues getting the site actually working in anything lower than IE11 so I can't be certain, but it worked fine in IE11.

Reply:
It's an unsafe transformation (meaning there are funny case that emit more code than expected), so I don't have it enabled.

Reply:
Hmm okay damn. They're very useful to have.

I'll change them.